src/aap_eda/core/models/activation.py

```diff
@@ -74,6 +80,46 @@ class Meta:
     status_updated_at = models.DateTimeField(null=True)
     status_message = models.TextField(null=True, default=None)
 
+    def save(self, *args, **kwargs):
```

Comment:
This is an important restriction on the model and it should have their own tests to ensure the expected behavior.